README.md

```diff
@@ -2,22 +2,95 @@
 
 [![npm version](https://img.shields.io/npm/v/vega-embed.svg)](https://www.npmjs.com/package/vega-embed)
 
-The [vega-embed](http://github.com/vega/vega-embed) module provides advanced support for embedding interactive Vega views into web pages. The primary features include:
+The [vega-embed](http://github.com/vega/vega-embed) module provides advanced support for embedding interactive Vega views into web pages. Current version supports only Vega 3 / Vega-Lite 2. The primary features include:
 
-- Load Vega specs from source text, parsed JSON, or URLs.
+- Load Vega/Vega-Lite specs from source text, parsed JSON, or URLs.
 - Add action links such as "View Source" and "Open in Vega Editor".
-- Parameterize visualizations with auto-generated dynamic query widgets.
 
-This last feature provides a powerful and convenient way to interact with a visualization by adding interactive widgets such as sliders, text fields, dropdown menus, and radio boxes. For Vega specifications without interactive signals defined, vega-embed provides _rewriting rules_ for injecting interactivity into an existing spec.
 
-For more, see the [documentation on the Vega wiki](http://github.com/vega/vega/wiki/Embed-Vega-Web-Components).
+_As Vega 3's `signal` supports [bind](https://github.com/vega/vega/blob/master/PORTING_GUIDE.md#scales), `parameter` property in the older version is now deprecated._
+<!--TODO : Link the older version document -->
+
+## Basic Example
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <!-- Please import Vega 3 & Vega-Lite 2 js  -->
+  <script src="vega.js" charset="utf-8"></script>
+  <script src="vega-lite.js" charset="utf-8"></script>
+  <script src="vega-embed.js" charset="utf-8"></script>
+</head>
+<body>
+
+<div id="vis"></div>
+
+<script type="text/javascript">
+  var embed_spec = {
```

Comment:
Sorry, I might've missed discussion on this but I disagree. If we keep vega-embed around, I do not think embed specs should be removed in favour of alternative ways of invoking the API. 

An embed specs affords the same advantages as the rest of our specification languages. For example, I can load, modify, and iterate on an embed spec as part of the editor infrastructure (rather than loading alternate panes to modify just the parameters, etc.).